src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/Create/CreateCommandHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Commands.Persistence;
+using SFA.DAS.EmployerIncentives.Commands.Types.ApprenticeshipIncentive;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.ValueTypes;
+using SFA.DAS.EmployerIncentives.Domain.Factories;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.Create
+{
+    public class CreateCommandHandler : ICommandHandler<CreateCommand>
+    {
+        private readonly IIncentiveApplicationDomainRepository _applicationDomainRepository;
+        private readonly IApprenticeshipIncentiveFactory _apprenticeshipIncentiveFactory;
+        private readonly IApprenticeshipIncentiveDomainRepository _apprenticeshipIncentiveDomainRepository;
+        
+
+        public CreateCommandHandler(
```

Comment:
Given we're duplicating data because of the different bounded context I think we need to have a greater distinction between bounded contexts (maybe a new db, new projects etc) as it's going to be difficult to understand what each data item is for/

Reply:
Yep.  We put the new bounded context into a separate namespace/folder but separate db and projects sounds like a better approach.

Reply:
I've created a tech debt ticket (https://skillsfundingagency.atlassian.net/browse/EI-417) to split out the projects but @Pauljgraham is going to create a schema in the db for the tables as a starting point

Reply:
We did wonder about using a different schema in the db rather than dbo.  But that doesn't seem to be common practice here.